components/autocomplete/events.md

```diff
@@ -227,6 +228,62 @@ When using `OnRead`, make sure to set `TItem` and `TValue`.
 }
 ````
 
+## OnItemRender
+
+The `OnItemRender` event fires when each item in the collection passed to the `Data` parameter of the AutoComplete renders. 
```

Comment:
Consider shorter and simpler sentences. This one contains a very long subordinate clause that separates two related words - "item" and "renders". It's also more technical that necessary. For example:

* The `OnItemRender` event fires when each item in the `Data` collection renders.
* The `OnItemRender` event fires when each item in the AutoComplete dropdown renders.

It's also worth clarifying if the event fires every time the dropdown is opened or not. What about filtering scenarios, when the visible items change all the time?

Reply:
It will fire again if the items in the popup are re-rendered (e.g. Filtering is used). So it truly fires each time an item renders.

On the topic of the sentence, I have noticed that we are using longer sentences when explaining things in the docs. I totally agree with you that this can be shortened by removing the subordinate clause.